qemu/tests/cgroup.py

```diff
@@ -194,7 +194,7 @@ def param_add_scsi_disks():
             process.system("echo 1 > /sys/bus/pseudo/drivers/scsi_debug/"
                            "add_host", shell=True)
             time.sleep(1)   # Wait for device init
-            dev = process.system_output("ls /dev/sd* | tail -n 1", shell=True)
+            dev = process.getoutput("ls /dev/sd* | tail -n 1", shell=True).stdout_text.strip()
```

Comment:
`.stdout_text.strip()` in not necessary here.

Reply:
updated